.devops/code-review-pipelines.yml

```diff
@@ -0,0 +1,106 @@
+parameters:
```

Comment:
@aomegax @jacopocarlini some things to remember, in to next pipeline improve : 
- could be better use jobs for [timeout](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?tabs=yaml&view=azure-devops#timeouts)
- add [cron trigger](https://docs.microsoft.com/it-it/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml#scheduled-triggers)